5.rb

```diff
@@ -0,0 +1,19 @@
+def remove_first_last_two_char(str)
+    result = ""
+    len = str.length
+
+    if len <= 4
+        return result
+        
+    # use pos + .each as .each_with_index
+    pos = 0
+    str.split('').each do 
+        |c|
```

Comment:
argument of block should be at the same line
```suggestion
    str.split('').each do |c|
```